### PR TITLE
Portal Person MGMT into spaceship 👩‍👩‍👦

### PR DESCRIPTION
**Pull request by hjanuschka**
adds support for person mgmt on portal via spaceship.

as requested in
  * https://github.com/fastlane/fastlane/issues/7903

related:
 * https://github.com/fastlane/fastlane/pull/7993

bildschirmfoto 2017-01-30 um 10 59 17

# API

## List all persons:
(returns array of `Spaceship::Portal::Person`

```ruby
Spaceship::Portal::Persons.all
```

## Find by email:

```ruby
person = Spaceship::Portal::Persons.find("helmut@januschka.com")
```

## Create/Invite a person

```ruby
Spaceship::Portal::Persons.invite("helmut1@januschka.com", "admin")
```

## Delete a person

```ruby
person = Spaceship::Portal::Persons.find("helmut@januschka.com")
person.remove!
```

## Change role of member.

```ruby
person = Spaceship::Portal::Persons.find("helmut@januschka.com")
person.change_role("member")
```

### CODE REVIEW

**Comment by KrauseFx** [comment]
Looks cool, what do you want to use this for?

**Comment by hjanuschka** [comment]
@KrauseFx  added specs and docs, pr is ready for review 👍 

one use case could be to disable admins automatically once they don't need to be on team anymore.
overall it think its nice to manage team members via spaceship. (as the tunes member part is already in spaceship, portal is just a logical step? )

File: spaceship/spec/portal/persons_spec.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+describe Spaceship::Portal::Persons do
+  before { Spaceship.login }
+  let(:client) { Spaceship::Portal::App.client }
+  it "Should load all persons" do
```
Comment (KrauseFx) [inline spaceship/spec/portal/persons_spec.rb]: Usually I don't think we have 2 separate files for `person` and `persons`, any reason why those are separate?

File: spaceship/spec/portal/persons_spec.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    all = Spaceship::Portal::Persons.all
+    expect(all.length).to eq(3)
+  end
+  it "Should find a specific person" do
+    person = Spaceship::Portal::Persons.find("helmut@januschka.com")
+    expect(person.email_address).to eq("helmut@januschka.com")
+  end
+  it "should invite a new one" do
+    expect { Spaceship::Portal::Persons.invite("helmut@januschka.com", "admin") }.to_not raise_error
```
Comment (KrauseFx) [inline spaceship/spec/portal/persons_spec.rb]: Mhhh, that doesn't really test anything, can you ensure the correct request is sent instead?

Comment (KrauseFx) [inline spaceship/spec/portal/persons_spec.rb]: something like `expect(Spaceship::Client).to receive(:...)`

**Review by KrauseFx — COMMENTED** [review]
Looks great already 👍

**Comment by hjanuschka** [comment]
@KrauseFx  tests improved, i did it in 2 files as i thought one class per file  - and one spec per class?

you want me to combine the specs? and/or the implemenation itself?

i did the exact same with the tunes PR. (having member_spec, and members_spec)

**Review by KrauseFx — APPROVED** [review]

**Pull request merged**